ROMFS/px4fmu_common/init.d-posix/px4-rc.simulator

```diff
@@ -118,6 +118,9 @@ elif [ "$PX4_SIMULATOR" = "gz" ] || [ "$(param show -q SIM_GZ_EN)" = "1" ]; then
 			echo "ERROR [init] gz_bridge failed to start and spawn model"
 			exit 1
 		fi
+		sleep 1 # Needed to wait for the model to spawn
+		gz service -s /gui/follow --reqtype gz.msgs.StringMsg --reptype gz.msgs.Boolean --timeout 2000 --req "data: \"${PX4_SIM_MODEL#*gz_}_0\""
```

Comment:
What do you think about pushing this into the gz_bridge module so that we don't need sleep in init?

Reply:
Done!